src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -0,0 +1,53 @@
+
+## Problem
+
+You want to compile and/or run different test or binary targets using different JVMs.
+
+Or, you need to specify command-line options globally for the JVM on a per-JVM version basis. For
+example, using `--add-opens` flags, but only when running on JVMs that support it.
+
+## Solution
+
+Declare JVM platforms using the `jvm-platform` configuration section to define the set of available
+platforms.
+
+On targets, add the kwarg `runtime_platform` set to the name of one of the platforms defined in
+`jvm-platform`. That will tell pants to use that platform when running that target in a JVM.
+
+To compile with a specific platform, set the `platform` on the target.
+
+
+## Discussion
+
```

Comment:
An introductory sentence here would be good, to signal that "we're going to talk about what you can specify in the global config now". Something like:

```suggestion
Here are the different configurations that you can set when defining a JVM platform in your global configuration.

```